gen-resourcesdocs/config/v1.21/fields.yaml

```diff
@@ -0,0 +1,693 @@
+- definition: io.k8s.api.core.v1.PodSpec
+  field_categories:
+  - name: Containers
+    fields:
+    - containers
+    - initContainers
+    - imagePullSecrets
+    - enableServiceLinks
+  - name: Volumes
+    fields:
+    - volumes
+  - name: Scheduling
```

Comment:
Are some `definitions` in this file directly generated from the properties in the spec? It is nice to have the categories on the page to break up the to create the fields but I am wondering how to automatically maintain the configuration files.